src/ui/.env.local

```diff
@@ -1 +0,0 @@
-NEXT_PUBLIC_JINJAT_URL=http://127.0.0.1:8581
```

Comment:
let's not edit file

Reply:
So are there any other ways to connect the public URL of the server to the app instead of the self-hosted URL without editing this file?